### PR TITLE
WID-543 - Add X-Client-Properties to default headers sent with the SearchAPIClient

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- Add X-Client-Properties to default headers sent with the SearchAPIClient
---
Ticket: https://jira.uitdatabank.be/browse/WID-543

### CODE REVIEW

File: app/SearchAPI/SearchAPIServiceProvider.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ public function register(Container $pimple)
                     'base_uri' => $pimple['search_api.base_url'],
                     'headers' => [
                         'X-Api-Key' => $pimple['search_api.api_key'],
+                        'X-Client-Properties' => 'cluster|widgets, cluster|snowplow',
```
Comment (brampauwelyn) [inline app/SearchAPI/SearchAPIServiceProvider.php:23]: It probably doesn't matter but in the ticket it is written in small letters.
`x-client-properties`

**Review by brampauwelyn — APPROVED** [review]
Cool!

File: app/SearchAPI/SearchAPIServiceProvider.php (hunk 1 above)
Comment (simon-debruijn) [reply app/SearchAPI/SearchAPIServiceProvider.php:23]: An HTTP header consists of its case-insensitive name

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

I matched it with the one above to be consistent 😃

Comment (brampauwelyn) [reply app/SearchAPI/SearchAPIServiceProvider.php:23]: 😂

**Pull request merged**